### DISCUSSION TITLE
rancher is failed showing CrashLoopBackOff

### DISCUSSION THREAD

**Discussion by bittu664** [mysticaltech/terraform-hcloud-kube-hetzner#731] [Q&A] [answered]
i created my new k3s cluster using terraform and this is the version:- v1.26.3+k3s1

<img width="1312" alt="Screenshot 2023-04-17 at 6 07 44 PM" src="https://user-images.githubusercontent.com/28609215/232486189-2b81f5b7-7c19-4cde-900e-afafdd50c553.png">

but for some reason rancher is failing its shows 
<img width="1078" alt="Screenshot 2023-04-17 at 5 22 58 PM" src="https://user-images.githubusercontent.com/28609215/232485910-b3102b44-87a1-4af3-a977-a3bb601d10c6.png">

**Comment by ifeulner** [comment, marked as answer]
Clear message, Rancher does not work with kubernetes v1.26 yet.

**Reply by bittu664** [reply]
okay
